sdk/spring/azure-identity-spring/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```

Comment:
`spring-cloud-dependencies:2.5.4` -> `spring-cloud-dependencies:2021.0.0`

Reply:
Could you please add `This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1`